hack/kfc-dev.test.js

```diff
@@ -0,0 +1,85 @@
+const { expect } = require("@jest/globals");
+const { execSync } = require("node:child_process");
+const fs = require("fs");
+const path = require("path");
+const { beforeEach } = require("node:test");
+
+describe("KFC Dev Helper E2E Test", () => {
+  beforeEach(() => {
+    try {
+      fs.rmSync("kubernetes-fluent-client", { recursive: true, force: true });
+      execSync("docker rmi pepr");
```

Comment:
Is this actually doing what you want?  I think you have to specify the full tag / image id in order to actually delete an image... at least on my machine?

Maybe that `catch (err)` below is squashing your error?

![image](https://github.com/user-attachments/assets/83c1309f-0de4-4ab6-9a4d-ef754a9a940c)